OracleDatabase/SingleInstance/README.md

```diff
@@ -456,6 +498,79 @@ Once the container has been started you can connect to it just like to any other
     sqlplus sys/<your password>@//localhost:1521/XE as sysdba
     sqlplus system/<your password>@//localhost:1521/XE
 
+### Running Oracle True Cache in a container (Supported from version 23.4.0 onwards)
+
+Oracle True Cache is an in-memory, consistent, and automatically managed cache for Oracle Database.
```

Comment:
Linking to https://docs.oracle.com/en/database/oracle/oracle-database/23/odbtc/overview-oracle-true-cache.html would be helpful for users.

Reply:
Added. Thanks